src/mixins/embed-button.js

```diff
@@ -49,6 +50,11 @@ export default {
     };
   },
   async mounted() {
+    window.top.addEventListener('message', this.handleWindowMessage);
+    // Notify app when button is mounted
+    if (window.top.ReactNativeWebView) {
+      window.top.ReactNativeWebView.postMessage(JSON.stringify({ action: 'MOUNTED' }));
+    }
```

Comment:
We should wrap these in try catch in case the iframe is sandboxed, in which `window.top.addEventListener` would throw an error